vm.exceptionBreakpoint("java.lang.Exception") { e ->
  println "caught exception:"
  e.dumpStackTrace(System.out);
}